Scenarios/Controllers/HighCpuController.cs

```diff
@@ -12,12 +13,17 @@ public class HighCpuController : Controller
         [HttpPost("/cpu-1")]
         public string BurnCpu()
         {
-            var spin = new System.Threading.SpinWait();
-            for (int i = 0; i < 100000; i++)
+            Stopwatch watch = new Stopwatch();
+            watch.Start();
+
+            while (true)
             {
-                spin.SpinOnce();
+                watch.Stop();
+                if (watch.ElapsedMilliseconds > 1000 * 5)
+                    break;
+                watch.Start();
             }
-            return "Done";
+            return "Working";
```

Comment:
@davidfowl If you have some time left, may you explain why you changed that? 😉

Is this because `SpinOnce` 100000 times took to long or because you want to see a spike in CPU performance?
Would `SpinWait.SpinUntil` be an option here too?

Thanks!